typing_extensions/src_py3/typing_extensions.py

```diff
@@ -981,8 +981,8 @@ def __new__(cls, *args, **kwds):
             if OLD_GENERICS:
                 return _generic_new(_next_in_mro(cls), cls, *args, **kwds)
             return _generic_new(cls.__next_in_mro__, cls, *args, **kwds)
-
-    Protocol.__doc__ = Protocol.__doc__.format(bases="Protocol, Generic[T]" if
+    if Protocol.__doc__ is not None:
+        Protocol.__doc__ = Protocol.__doc__.format(bases="Protocol, Generic[T]" if
                                                OLD_GENERICS else "Protocol[T]")
```

Comment:
You should also add four spaces of indent here.

Reply:
Thank you! C: should I commit a second time (or just re-write the initial commit)?